cmd/juju/cloud/detectcredentials.go

```diff
@@ -490,17 +497,15 @@ func (c *detectCredentialsCommand) addRemoteCredentials(ctxt *cmd.Context, cloud
 			result, err := client.UpdateCloudsCredentials(verified)
 			if err != nil {
 				logger.Errorf("%v", err)
-				ctxt.Warningf("Could not add credentials remotely, on controller %q", c.ControllerName)
+				ctxt.Warningf("Could not upload credentials to controller %q", c.ControllerName)
```

Comment:
"credential"
this deals with one at a time right?

Reply:
No, this deals with per-cloud credentials.. could be more than one.